src/renderer/Default.js

```diff
@@ -98,6 +98,7 @@ export default function Default() {
                     <Route path="/" exact render={props => <Dashboard {...props} />} />
                     <Route path="/settings" render={props => <Settings {...props} />} />
                     <Route path="/accounts" render={props => <Accounts {...props} />} />
+                    <Redirect from="/manager/reload" to="manager" />
```

Comment:
i've seen other suggest history.go(0).
I don't have opinion

Reply:
history.go(0) reloads the whole app. Not possible to use